cms/templates/settings.html

```diff
@@ -199,6 +199,26 @@ <h3 class="title">${_("These Dates Are Not Used When Promoting Your Course")}</h
           </div>
           % endif
         </section>
+
+        <section class="group-settings details">
```

Comment:
Yes, I believe that it should show regardless of whether or not the course is running on edx Drupal.